packages/components/rate/src/rate.vue

```diff
@@ -88,10 +89,17 @@ export default defineComponent({
   setup(props, { emit }) {
     const elForm = inject(elFormKey, {} as ElFormContext)
 
+    const rateSize = useSize()
     const currentValue = ref(props.modelValue)
     const hoverIndex = ref(-1)
     const pointerAtLeftHalf = ref(true)
 
+    const prefix = 'el-rate'
+    const rateKls = computed(() => [
+      prefix,
+      rateSize.value ? `${prefix}--${rateSize.value}` : '',
```

Comment:
So this `rateSize.value` will be always truthy, right? I wonder if we still need this check.

Reply:
Looks like so, I think we can pass the checking part on sizes, it always fallbacks to `default`.